test/avocado/selenium-machines-nonroot.py

```diff
@@ -0,0 +1,152 @@
+from testlib_avocado.machineslib import MachinesLib
+from testlib_avocado.seleniumlib import clickable, invisible, text_in
+from testlib_avocado.timeoutlib import wait
+
+
+class MachinesNonrootTestSuite(MachinesLib):
+    '''
+    :avocado: enable
+    :avocado: tags=machines_n
+    '''
+
+    def operationsForVm(self, vm_args):
+        name = vm_args.get('name')
+        log_file = vm_args.get('logfile')
+
+        if log_file and name and 'vnc' in vm_args.get('graphics'):
+            self.wait_vm_complete_start(vm_args)
+            self.click(self.wait_css('#vm-{}-reboot'.format(name), cond=clickable))
+            wait(lambda: self.machine.execute("sudo cat {} | grep 'Sent SIGTERM to all processes' | wc -l".format(log_file)).strip() == '1',
+                 delay=3)
+
+            self.wait_vm_complete_start(vm_args)
+            self.machine.execute('sudo sh -c "echo > {}"'.format(log_file))
+            self.click(self.wait_css('#vm-{}-reboot-caret'.format(name), cond=clickable))
+            self.click(self.wait_css('#vm-{}-forceReboot'.format(name), cond=clickable))
+            wait(lambda: "Initializing cgroup subsys cpuset" in self.machine.execute("sudo cat {}".format(log_file)),
+                 delay=3)
+
+            self.wait_vm_complete_start(vm_args)
+            self.click(self.wait_css('#vm-{}-off-caret'.format(name), cond=clickable))
+            self.click(self.wait_css('#vm-{}-sendNMI'.format(name), cond=clickable))
+            wait(lambda: "NMI received" in self.machine.execute("sudo cat {}".format(log_file)),
+                 delay=3)
+
+            self.click(self.wait_css('#vm-{}-off'.format(name), cond=clickable))
+            self.wait_css('#vm-{}-off'.format(name), cond=invisible)
+            self.wait_css('#vm-{}-run'.format(name))
+
+            self.machine.execute('sudo sh -c "echo > {}"'.format(log_file))
+            self.click(self.wait_css('#vm-{}-run'.format(name), cond=clickable))
+            self.wait_css('#vm-{}-run'.format(name), cond=invisible)
+            self.wait_css('#vm-{}-off'.format(name))
+
+            self.click(self.wait_css('#vm-{}-off-caret'.format(name), cond=clickable))
+            self.click(self.wait_css('#vm-{}-forceOff'.format(name), cond=clickable))
+            self.wait_css('#vm-{}-off'.format(name), cond=invisible)
+            self.wait_css('#vm-{}-run'.format(name))
+
+    def testNoneRootOperationWithVm(self):
```

Comment:
You mean NonRoot, no NoneRoot I guess :)

Reply:
Yes, you are right, fixed in the commit